recipes/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "pyincore" %}
+{% set version = "1.0.0" %}
+ 
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+ 
+source:
+  path: /path/to/pyincore
+  # PR: https://opensource.ncsa.illinois.edu/bitbucket/projects/INCORE1/repos/pyincore/pull-requests/12/overview
+ 
+build:
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  #number: 1
+  noarch: python
+  script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed -vv . " # verbose
+ 
+requirements:
+  build:
+    - python>=3.6
+    - pip
+    - numpy 1.16
+ 
+  host:
+    - python>=3.6
+    - pip
+    - numpy 1.16
+     
+  run:
+    - python>=3.6
+    - {{ pin_compatible('numpy') }}
+    - boto3
+    - deprecated
+    - fiona>=1.8.4
+    - geopandas>=0.6.1
+    - jsonpickle>=1.1
+    - libspatialindex>=1.9.0
+    - networkx>=2.2
+    - numpy>=1.16.1
+    - pytest>=3.9.0
+    - python-jose>=3.0
+    - pyproj>=1.9.6
+    - pyyaml>=3.13
+    - pycodestyle>=2.6.0
+    - pyomo>=5.6
+    - owslib>=0.17.1
+    - pandas>=0.24.1
+    - rasterio>=1.0.18
+    - requests>=2.21.0
+    - rtree>=0.8.3
+    - scipy>=1.2.0
+    - shapely>=1.6.4.post1
+    - wntr>=0.1.6
+    - ipopt>=3.11
+ 
+test:
+  # Python imports
+  imports:
+    - pyincore
+    - pyomo
+   
+  requires:
+    # Put any additional test requirements here.  For example
+    - pytest>=3.9
```

Comment:
Should this be 3.8?

Reply:
is 3.8 okay? If so, I will modify it

Reply:
I'm just trying to understand why we set 3.9 here. Have we been using 3.9 in the conda recipe previously?

Reply:
We've been using 3.6 so far

Reply:
Just going to leave this year, we realized this is the pytest version not python version so this comment can be ignored.